README.md

```diff
@@ -225,12 +225,12 @@ defp deps do
 
 Sparrow suports many pools in a single `:sparrow` instance.
 [Tags](#tags) are used to choose a pool when sending a notification.
-[](https://placehold.it/15/ff0000/ff0000?text=+) *`This feature is not yet implemented for FCM.`*
+
 
 ## Tags
 
 Tags is a mechanism allowing to choose a pool to send a notification from.
-Each pool has a defined list of tags (`[]` as default).
+Each pool has a defined list of tags (`[]` as default in `APNS` and `[account]` as default in `FCM`, where account is extracted from `FCM` authentication JSON file).
```

Comment:
Why do we make a non-empty default value? How is that used?

Reply:
We wanted to be able to choose a pool based on our account. Doing it explicitly makes less sense because then the user has to pass both FCM token and one of the values inside as a tag. Additionally, if the user is not interested in connecting accounts to pools it doesn't really change anything since this tag will simply be unused

Reply:
But it looks weird, that we do that for FCM but with APNS topics, we don't. We should either do that for both or for none IMHO.

Reply:
Also, I kind doubt users would like to use account name as a tag - this is pretty sensitive data.

Reply:
Ok, changed to [] as default - Can't quite remember the exact reasoning behind that